jupyter_server/event-schemas/contentsmanager-actions/v1.yaml

```diff
@@ -0,0 +1,83 @@
+"$id": eventlogging.jupyter.org/jupyter_server/contentsmanager-actions
+version: 1
+title: Contents Manager activities
+personal-data: true
+description: |
+    Record actions on files via the ContentsManager REST API.
+
+    The notebook ContentsManager REST API is used by all frontends to retreive,
+    save, list, delete and perform other actions on notebooks, directories,
+    and other files through the UI. This is pluggable - the default acts on
+    the file system, but can be replaced with a different ContentsManager
+    implementation - to work on S3, Postgres, other object stores, etc.
+    The events get recorded regardless of the ContentsManager implementation
+    being used.
+
+    Limitations:
+
+    1. This does not record all filesystem access, just the ones that happen
+       explicitly via the notebook server's REST API. Users can (and often do)
+       trivially access the filesystem in many other ways (such as `open()` calls
+       in their code), so this is usually never a complete record.
+    2. As with all events recorded by the notebook server, users most likely
+       have the ability to modify the code of the notebook server. Unless other
+       security measures are in place, these events should be treated as user
+       controlled and not used in high security areas.
+    3. Events are only recorded when an action succeeds.
+type: object
+required:
+- action
+- path
+properties:
+  action:
+    enum:
+    - get
+    - create
+    - save
+    - upload
+    - rename
+    - copy
+    - delete
+    category: unrestricted
+    description: |
+        Action performed by the ContentsManager API.
+
+        This is a required field.
+
+        Possible values:
+
+        1. get
+           Get contents of a particular file, or list contents of a directory.
+
+        2. create
+           Create a new directory or file at 'path'. Currently, name of the
+           file or directory is auto generated by the ContentsManager implementation.
+
+        3. save
+           Save a file at path with contents from the client
+
+        4. upload
+           Upload a file at given path with contents from the client
+
+        5. rename
+           Rename a file or directory from value in source_path to
+           value in path.
+
+        5. copy
+           Copy a file or directory from value in source_path to
+           value in path.
+
+        6. delete
+           Delete a file or empty directory at given path
+  path:
+    category: personally-identifiable-information
```

Comment:
We decided to use URIs for category filtering, and there are a few well-known ones defined in https://jupyter-telemetry.readthedocs.io/en/latest/pages/schemas.html#property-categories.

Reply:
This requires changes in jupyter/telemetry master branch that haven't been made a released yet. Should we make a new release for telemetry?

Reply:
Yeah, making a new release there seems appropriate. Would you be able to do that?

Reply:
Nope I don't have the permission.

Reply:
I have updated this one and use the latest commit on jupyter/telemetry master for now, will change it once we have a new release.

Also should the `unrestricted` category be URI or just a generic one? Seems like we're currently using a universal `unrestricted` category
https://github.com/jupyter/telemetry/blob/d2c19ffa03fde7d1903759de4a265ef711b333ef/jupyter_telemetry/eventlog.py#L165-L176

While we're on the topic of URI may I have your thoughts on jupyter/telemetry#56? Thanks!